Src/yaml/Makefile.am

```diff
@@ -0,0 +1,4 @@
+AM_CPPFLAGS = -I$(top_srcdir)/include -Wall
```

Comment:
I am generally confused why there are so many YAML files. Are these Makefiles even being used?

Reply:
I've tidied up the YAML folder, you're right it had a lot of cruft.